blocks/columns/columns.js

```diff
@@ -1,16 +1,37 @@
 export default function decorate(block) {
   const cols = [...block.firstElementChild.children];
   block.classList.add(`columns-${cols.length}-cols`);
+  const imgColClass = 'columns-img-col';
+
+  const isTeaser = block.classList.contains('teaser');
+
+  // teaser is also a variation of thirds
+  if (isTeaser) {
+    block.classList.add('thirds');
+  }
 
   // setup image columns
   [...block.children].forEach((row) => {
     [...row.children].forEach((col) => {
       const pic = col.querySelector('picture');
       if (pic) {
         const picWrapper = pic.closest('div');
-        if (picWrapper && picWrapper.children.length === 1) {
+        // less than or equals to two in case image is wrapped with <a> tag
+        if (picWrapper && picWrapper.children.length <= 2) {
           // picture is only content in column
-          picWrapper.classList.add('columns-img-col');
+          picWrapper.classList.add(imgColClass);
+        }
+
+        if (isTeaser) {
+          const teaserWrapper = document.createElement('div');
+          teaserWrapper.classList.add('teaser-wrapper');
+          ['style-a', 'style-b', 'style-c'].forEach((style) => {
+            const bubble = document.createElement('i');
```

Comment:
instead of an `i` tag, I'd suggest a `span` and we should also probably set `aria-hidden` on them. (if there were 2, then I'd probably use pseudo-elements, but with 3 that is harder to do).